db_master_deploy/deploy.sh

```diff
@@ -11,10 +11,11 @@ display_usage() {
     echo -e "Usage:\n$0 -s source_objects -t target_staging -a timestamp for BOD snapshot/archive (YYYYMMDD)"
     echo -e "\t-s comma delimited list of source databases and/or tables - mandatory"
     echo -e "\t-t target staging - mandatory choose one of '${targets}'"
+    echo -e "\t-r materialized view to refresh - Optional. You can define (with dbname.schema.tablename !!) one or more materialized views to refresh'"
```

Comment:
Global on/off switch would be my preference for now. default off.

If we use the no-read-lock approach, it's easiest to enable it again (intermediate solution).